lib/flip_card.dart

```diff
@@ -155,6 +157,10 @@ class FlipCardState extends State<FlipCard>
     ).animate(controller!);
 
     widget.controller?.state = this;
+
+    if (widget.autoFlipDuration != null) {
+      Future.delayed(widget.autoFlipDuration!, () => toggleCard());
```

Comment:
You can also just pass `toggleCard` directly as the second parameter (without creating an anonymous function to call it yourself)

Reply:
I updated it.